cellpack/test-recipes/v2/nested.json

```diff
@@ -1,6 +1,6 @@
 {
-    "version": "1.0.0",
-    "format_version": "2.0.0",
+    "version": "1.0",
```

Comment:
versions here should still be 3 digits. we're just changing the `format_version`